specs/derivation.md

```diff
@@ -679,14 +679,14 @@ To interact with the engine, the [execution engine API][exec-engine] is used, wi
 
 [exec-engine]: exec-engine.md
 
```

Comment:
Perhaps a note in here that a backwards compatible change was made would be nice to have